.gitignore

```diff
@@ -0,0 +1,107 @@
+node_modules/
+config.env
+
```

Comment:
Cleaned it, the rest is standerd for everyproject. They don't exist because they are ignored.